src/test/regress/expected/alter_database_owner.out

```diff
@@ -0,0 +1,240 @@
+CREATE SCHEMA alter_database_owner;
+SET search_path TO alter_database_owner, public;
+CREATE USER database_owner_1;
```

Comment:
wouldn't this test fail on enterprise?

Reply:
Good catch, I think the output might be different for the creation of the user and the creation via `run_command_on_workers` I will have a look at comparable test suites, how they compensate for that.

I can imagine we will
 - remove the `run_command_on_workers` call
 - remove the warnings from the output file

But will look into the differences of other tests that create users in community.

Reply:
as an example see: (needs to be run from enterprise repo)
```
git diff open-master..origin/enterprise-master  -- src/test/regress/sql/aggregate_support.sql
```

We have removed the `run_comman_on_workers` for the user creation in the enterprise version of the test. Once we approve this PR I will squash all commits, then open a PR with the squashed commit + a commit to make tests pass on enterprise. We can then merge this PR via a merge commit as-well as the PR on enterprise. This will keep the lineage of commits while directly fixing the tests on enterprise.